db/c.cc

```diff
@@ -5632,6 +5673,33 @@ void rocksdb_options_set_min_level_to_compress(rocksdb_options_t* opt,
   }
 }
 
+rocksdb_livefiles_t* rocksdb_livefiles_create() {
+  return new rocksdb_livefiles_t;
+}
+
+void rocksdb_livefiles_add(rocksdb_livefiles_t* lf,
+                           const char* column_family_name, const char* name,
+                           const char* directory, size_t size, int level,
+                           const char* start_key, size_t start_key_len,
+                           const char* end_key, size_t end_key_len,
+                           uint64_t smallest_seqno, uint64_t largest_seqno,
+                           uint64_t num_entries, uint64_t num_deletions) {
```

Comment:
What's a better way to do this that is still easy enough to use from Rust? I could define a struct for it; my original approach was to wrap the underlying struct but the Rust create has its own `LiveFile` which isn't directly related to the C struct, and I wanted to reuse that as much as possible.